code/stable/gamephysics/SRS/GamePhysics_SRS.tex

```diff
@@ -994,13 +987,11 @@ \subsubsection{Data Definitions}
               \item{$\mathbf{v}$ is the velocity ($\frac{\text{m}}{\text{s}}$)}
               \end{symbDescription}
 \\ \midrule \\
-Notes & The kinetic energy of an object is the energy it possess due to its motion.
-        
-        \hyperref[assumpOT]{A: objectTy}
+Notes & Kinetic energy is the measure of the energy a body possesses due to its motion.
         
-        \hyperref[assumpOD]{A: objectDimension}
+        All bodies are assumed to be rigid (from \hyperref[assumpOT]{A: objectTy}) and two-dimensional (from \hyperref[assumpOD]{A: objectDimension}).
```

Comment:
We can keep these assumptions, although kinetic energy is already separate from strain energy.  We probably do not need the rigid body assumption, but I don't have time right now to think about this too deeply.